### ISSUE TITLE
[8.0] Domain filter on search view in field tag does not work

### ISSUE THREAD

**Issue by mariuszmizgier** [odoo/odoo#6116]
In V7 while defining search view you were able to set a domain parameter, ie on Quotations you were able to do it like this below: 
```xml
<field name="partner_id" domain="[('customer','=',True)]" filter_domain="[('partner_id', 'child_of', self)]"/>
```
When typing in search view a name of a company, suggested results were filtered to Customers only.
In V8 when you do the same thing, domain filter is not applied, after clicking on arrow on Search Customers for: you will see all the records. 

I've uploaded a video recorded on runbot to show the difference. See it at https://www.youtube.com/watch?v=XPm-L1q7M2k

**Comment by gurneyalex** [comment]
I confirm I was able to reproduce this issue on runbot.

**Comment by nle-odoo** [comment]
Hello, PR #6355 should fix this in 8.0.

**Comment by gurneyalex** [comment]
thanks
